### ISSUE TITLE
Diagnosis for differential misclassification

### ISSUE THREAD

**Issue by chainsawriot** [groceryheist/misclassificationmodels#1]
- [ ] Sargan's J Test
- [ ] Stratified diagnostic odds ratios and heterogeneity test (e.g. `metafor`)

**Comment by groceryheist** [comment]
Yes we should do this. Currently trying to decide how much we should attempt to automate, for now I think adding such feature, at least in the documentation and as "suggested" packages will be great.
